### PR TITLE
Documentation improvement for Ceph External Cluster

### PR DESCRIPTION
**Pull request by alexcpn**
**Description of your changes:**

Added a StorageClass example for better document usability to test CephExternal cluster

Solves: None
Signed-off-by: Alex Punnen <alexcpn@gmail.com>

Documentation improvement for Ceph External Cluster

**Which issue is resolved by this Pull Request:**
Resolves # None

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

File: Documentation/ceph-cluster-crd.md [hunk 1]
```diff
@@ -985,6 +985,67 @@ If the previous section has not been completed, the Rook Operator will still ack
...
+
+In Ceph Cluster let us list the pools available
+
+```console
+rados df
+POOL_NAME     USED OBJECTS CLONES COPIES MISSING_ON_PRIMARY UNFOUND DEGRADED RD_OPS  RD WR_OPS  WR USED COMPR UNDER COMPR 
+replicated_2g  0 B       0      0      0                  0       0        0      0 0 B      0 0 B        0 B         0 B 
+```
+
+Here is an example StorageClass configuration that uses the 'replicated_2g' pool from the external cluster
```
Comment (leseb) [inline Documentation/ceph-cluster-crd.md:1008]:
```suggestion
Here is an example StorageClass configuration that uses the `replicated_2g` pool from the external cluster:
```

File: Documentation/ceph-cluster-crd.md [hunk 2]
```diff
@@ -985,6 +985,67 @@ If the previous section has not been completed, the Rook Operator will still ack
...
+kubectl get CephCluster -n rook-ceph-external
+NAME                 DATADIRHOSTPATH   MONCOUNT   AGE    STATE       HEALTH
+rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
+```
+
+Before you create a StorageClass with this cluster you will need to create a Pool in your external Ceph Cluster.
+
+#### Example StorageClass based on external Ceph Pool
+
+In Ceph Cluster let us list the pools available
```
Comment (leseb) [inline Documentation/ceph-cluster-crd.md:1000]:
```suggestion
In Ceph Cluster let us list the pools available:
```

File: Documentation/ceph-cluster-crd.md [hunk 3]
```diff
@@ -985,6 +985,67 @@ If the previous section has not been completed, the Rook Operator will still ack
...
+
+```console
+rados df
```
Comment (leseb) [inline Documentation/ceph-cluster-crd.md:1003]: I'd use `ceph osd lspools`

File: Documentation/ceph-cluster-crd.md [hunk 4]
```diff
@@ -985,6 +985,67 @@ If the previous section has not been completed, the Rook Operator will still ack
...
+    # in hyperconverged settings where the volume is mounted on the same node as the osds.
+    csi.storage.k8s.io/fstype: ext4
+
+# Delete the rbd volume when a PVC is deleted
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+EOF
+```
+
+You can now create persistant volume based on this StorageClass
```
Comment (leseb) [inline Documentation/ceph-cluster-crd.md:1047]:
```suggestion
You can now create a persistent volume based on this StorageClass.
```

File: Documentation/ceph-cluster-crd.md (hunk 3 above)
Comment (alexcpn) [reply Documentation/ceph-cluster-crd.md:1003]: Yes this is also fine; What I wanted was to check if the PV created was writing to external cluster, as last time I tested it was in rook-ceph and I thought all was well

After creation you will see a bump in size, and after some data is written
```
[root@ceph-1 ~]# ssh -i alex_ee.pem ceph-2 rados df
POOL_NAME        USED OBJECTS CLONES COPIES MISSING_ON_PRIMARY UNFOUND DEGRADED RD_OPS      RD WR_OPS     WR USED COMPR UNDER COMPR 
replicated_2g 151 MiB      27      0     81                  0       0        0    137 2.2 MiB     93 50 MiB        0 B         0 B 
```

Comment (leseb) [reply Documentation/ceph-cluster-crd.md:1003]: Works for me!

**Pull request merged**